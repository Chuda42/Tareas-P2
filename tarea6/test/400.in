# Implementar structs, las operaciones de crear y liberar, cantidadEnIterador
cantidadEnIterador
# comando que reinicia las estructruras (no es reiniciarIterador)
reiniciar
agregarAIterador 3
cantidadEnIterador
reiniciar
agregarAIterador 3
agregarAIterador 3
cantidadEnIterador
reiniciar
agregarAIterador 3
agregarAIterador 3
agregarAIterador 3
agregarAIterador 3
agregarAIterador 3
agregarAIterador 3
agregarAIterador 3
agregarAIterador 3
cantidadEnIterador
reiniciar
# Implementar enAlguno
enAlguno 0
enAlguno 1 4
enAlguno 2 4 8
enAlguno 5 1 3 7 8 9
reiniciar
agregarAIterador 2
enAlguno 0
enAlguno 1 4
enAlguno 2 4 8
enAlguno 5 1 3 7 8 9
enAlguno 1 2
reiniciar
agregarAIterador 4
agregarAIterador 7
agregarAIterador 9
enAlguno 0
enAlguno 1 4
enAlguno 2 4 8
enAlguno 5 1 3 7 8 9
enAlguno 6 1 3 7 8 9 11
reiniciar
# Implementar soloEnA. 
# El iterador 'a' es el parámetro 'it'. 
# El iterador 'b' es el que se lee desde la entrada estándar. 
soloEnA 0
soloEnA 1 4
soloEnA 2 4 8
soloEnA 5 1 3 7 8 9
reiniciar
agregarAIterador 2
soloEnA 0
soloEnA 1 4
soloEnA 2 4 8
soloEnA 5 1 3 7 8 9
soloEnA 1 2
reiniciar
agregarAIterador 4
agregarAIterador 7
agregarAIterador 9
soloEnA 0
soloEnA 1 4
soloEnA 2 4 8
soloEnA 5 1 3 7 8 9
soloEnA 6 1 4 7 8 9 11
Fin
