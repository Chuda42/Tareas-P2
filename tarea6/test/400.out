1>#  Implementar structs, las operaciones de crear y liberar, cantidadEnIterador.
2>0.
3>#  comando que reinicia las estructruras (no es reiniciarIterador).
4>Estructuras reiniciadas.
5>Agregando.
6>1.
7>Estructuras reiniciadas.
8>Agregando.
9>Agregando.
10>2.
11>Estructuras reiniciadas.
12>Agregando.
13>Agregando.
14>Agregando.
15>Agregando.
16>Agregando.
17>Agregando.
18>Agregando.
19>Agregando.
20>8.
21>Estructuras reiniciadas.
22>#  Implementar enAlguno.
23>
24>4 
25>4 8 
26>1 3 7 8 9 
27>Estructuras reiniciadas.
28>Agregando.
29>2 
30>2 4 
31>2 4 8 
32>1 2 3 7 8 9 
33>2 
34>Estructuras reiniciadas.
35>Agregando.
36>Agregando.
37>Agregando.
38>4 7 9 
39>4 7 9 
40>4 7 8 9 
41>1 3 4 7 8 9 
42>1 3 4 7 8 9 11 
43>Estructuras reiniciadas.
44>#  Implementar soloEnA. .
45>#  El iterador 'a' es el parámetro 'it'. .
46>#  El iterador 'b' es el que se lee desde la entrada estándar. .
47>
48>
49>
50>
51>Estructuras reiniciadas.
52>Agregando.
53>2 
54>2 
55>2 
56>2 
57>
58>Estructuras reiniciadas.
59>Agregando.
60>Agregando.
61>Agregando.
62>4 7 9 
63>7 9 
64>7 9 
65>4 
66>
67>Fin.
